### PR TITLE
[EC-366] Updated AndroidManifest format

### PR DESCRIPTION
**Pull request by fedemkr**
## Type of change
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [X] Other

## Objective

Updated AndroidManifest format as to what Visual Studio for Mac auto-formats to avoid merge conflicts/ resetting the file on commits.

## Code changes

* **AndroidManifest.xml:** Updated format of the file to what is automatically done by Visual Studio for Mac when the solution opens.

## Before you submit
- [X] I have checked for formatting errors (`dotnet tool run dotnet-format --check`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

File: src/Android/Properties/AndroidManifest.xml [hunk 1]
```diff
@@ -1,57 +1,48 @@
...
+			</intent-filter>
+		</activity>
+	</application>
+	<!-- Package visibility (for Android 11+) -->
+	<queries>
+		<intent>
+			<action android:name="*" />
+		</intent>
+	</queries>
+</manifest>
```
Comment (mp-bw) [inline src/Android/Properties/AndroidManifest.xml:48]: Will the removal of the trailing newline make dotnet format angry?

Comment (fedemkr) [reply src/Android/Properties/AndroidManifest.xml:48]: @mp-bw I've checked with `dotnet tool run dotnet-format --check` and it passes

Comment (mp-bw) [reply src/Android/Properties/AndroidManifest.xml:48]: @fedemkr Cool.  One last thing to check: When you edit the file in VS Win, does the auto formatting re-insert the newline?

Comment (fedemkr) [reply src/Android/Properties/AndroidManifest.xml:48]: @mp-bw not sure, but I think that nobody is using VS Win for mobile. I don't have an updated VS Win running here but I could set that up this week.

**Review by mp-bw — APPROVED** [review]

File: src/Android/Properties/AndroidManifest.xml (hunk 1 above)
Comment (mp-bw) [reply src/Android/Properties/AndroidManifest.xml:48]: @fedemkr You may be right - It was more common when I first joined.  Don't worry about testing it, we'll deal with it if/when it happens.

**Pull request merged**